packages/blade/src/components/Icons/EmailIcon/EmailIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactElement } from 'react';
+import type { IconComponent, IconProps } from '..';
+import useIconProps from '../useIconProps';
+import { Path, Svg } from '../_Svg';
+
+const EmailIcon: IconComponent = ({ size, color }: IconProps): ReactElement => {
+  const { height, width, iconColor } = useIconProps({ size, color });
+  return (
+    <Svg width={width} height={height} viewBox="0 0 22 18" fill="none">
```

Comment:
Can you pick the correct icon from Figma (root element) and use https://react-svgr.com/playground/?expandProps=none&native=true&svgo=false to convert it into a react component?

If its copied correctly, the viewBox would be `0 0 24 24`


<img width="352" alt="Screenshot 2022-11-22 at 3 05 04 PM" src="https://user-images.githubusercontent.com/24487274/203320678-9adb9ecb-a679-4218-8b85-cc6972fb54b0.png">

Reply:
Its root icon name is "mail". should i create name as MailIcon or EmailIcon is fine